doc/man/8/rbd-nbd.rst

```diff
@@ -45,6 +45,11 @@ Options
 
    Forbid writes by other clients.
 
+.. option:: --timeout
```

Comment:
Nit: same comment here (need ```*seconds*``` parameter)

Reply:
Check, I missed that too. I made a new commit, did a rebase and forced push to fork. Now it should be all good.

Reply:
... still missing on this line.

Reply:
(note: you should be able to just change the file, ```git add``` it, then ```git commit --amend``` it before a force push)